src/plugins/Tus10.js

```diff
@@ -46,12 +40,18 @@ export default class Tus10 extends Plugin {
           percentage = Math.round(percentage)
 
           // Dispatch progress event
-          this.core.emitter.emit('progress', {
-            plugin: this,
-            percentage: percentage
+          this.core.emitter.emit('upload-progress', {
```

Comment:
We could potentially destructure `emitter` and `log` from `this.core` if we have to call them a bunch, but it may be better to be explicit about using `this.core.emitter`, etc.

For instance:

`const { emitter, log } = this.core`